tests/models/seamless_m4t/test_feature_extraction_seamless_m4t.py

```diff
@@ -139,37 +139,59 @@ def test_feat_extract_to_json_file(self):
         dict_second = feat_extract_second.to_dict()
         self.assertEqual(dict_first, dict_second)
 
+    @require_torch
     def test_call(self):
```

Comment:
Let's do two separate test methods instead - `test_call_numpy` and `test_call_torch` - as the PT lines are all independent of the numpy ones

Reply:
Made changes to separate the test methods. Thanks!